sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py

```diff
@@ -0,0 +1,878 @@
+
+        :param name: The name of the certificate.
+        :type name: str
+        :returns: The updated certificate operation
+        :rtype: ~azure.security.keyvault.certificates._models.CertificateOperation
+        :raises:
+         :class:`KeyVaultErrorException<azure.keyvault.v7_0.models.KeyVaultErrorException>`
+        """
+        bundle = self._client.update_certificate_operation(
+            vault_base_url=self.vault_url,
+            certificate_name=name,
+            cancellation_requested=True,
+            **kwargs
+        )
+        return CertificateOperation._from_certificate_operation_bundle(certificate_operation_bundle=bundle)
+
+    @distributed_trace
+    def merge_certificate(
+        self, name, x509_certificates, enabled=None, not_before=None, expires=None, tags=None, **kwargs):
+        # type: (str, list[bytearray], Optional[bool], Optional[datetime], Optional[datetime]Optional[Dict[str, str]]) -> Certificate
+        """Merges a certificate or a certificate chain with a key pair existing on the server.
+
+        Performs the merging of a certificate or certificate chain with a key pair currently
+        available in the service. This operation requires the certificates/create permission.
+
+        :param name: The name of the certificate
+        :type name: str
+        :param x509_certificates: The certificate or the certificate chain to merge.
+        :type x509_certificates: list[bytearray]
+        :param enabled: Determines whether the object is enabled.
+        :type enabled: bool
+        :param not_before: Not before date of the secret in UTC
+        :type not_before: datetime.datetime
+        :param expires: Expiry date of the secret  in UTC.
+        :type expires: datetime.datetime
+        :param tags: Application specific metadata in the form of key-value pairs.
+        :type tags: dict[str, str]
+        :return: The merged certificate operation
+        :rtype: ~azure.security.keyvault.certificates._models.CertificateOperation
+        :raises:
+         :class:`KeyVaultErrorException<azure.keyvault.v7_0.models.KeyVaultErrorException>`
+        """
+        if enabled is not None or not_before is not None or expires is not None:
+            attributes = self._client.models.CertificateAttributes(
+                enabled=enabled, not_before=not_before, expires=expires
+            )
+        else:
+            attributes = None
+        bundle = self._client.merge_certificate(
+            vault_base_url=self.vault_url,
+            certificate_name=name,
+            x509_certificates=x509_certificates,
+            certificate_attributes=attributes,
+            tags=tags,
+            **kwargs
+        )
+        return CertificateOperation._from_certificate_operation_bundle(certificate_operation_bundle=bundle)
+
+    @distributed_trace
+    def get_pending_certificate_signing_request(self, name, custom_headers=None, **kwargs):
```

Comment:
I don't think I can do that because the get_certificate_operation returns a CertificateOperation and deserializes to that, but get_pending_certificate_signing_request just returns a string.